orchestrator/careplancontributor/webhook/eventhandler.go

```diff
@@ -1,16 +1,50 @@
 package webhook
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
 	events "github.com/SanteonNL/orca/orchestrator/careplancontributor/event"
+	"github.com/rs/zerolog/log"
 )
 
 var _ events.Handler = EventHandler{}
 
 type EventHandler struct {
-	URL string
+	client *http.Client
+	URL    string
+}
+
+func NewEventHandler(url string) EventHandler {
+	return EventHandler{URL: url, client: &http.Client{}}
 }
 
 func (w EventHandler) Handle(ctx context.Context, event events.Instance) error {
-	panic("implement me")
+	data, err := json.Marshal(event.FHIRResource)
+	if err != nil {
+		return fmt.Errorf("failed to serialize event: %w", err)
+	}
+
+	request, err := http.NewRequestWithContext(ctx, "POST", w.URL, bytes.NewReader(data))
+	if err != nil {
+		return fmt.Errorf("failed to create request: %w", err)
+	}
+
+	response, err := w.client.Do(request)
+	if err != nil {
+		return fmt.Errorf("failed to send request: %w", err)
+	}
+
+	defer response.Body.Close()
```

Comment:
we might need to drain the response body, not sure. Was previously required, but maybe not any more.

Reply:
After reading https://github.com/golang/go/issues/60240 I'm still not sure, let's do it anyways